operator/builtin/input/k8sevent/k8s_event.go

```diff
@@ -27,14 +27,19 @@ func init() {
 // NewK8sEventsConfig creates a default K8sEventsConfig
 func NewK8sEventsConfig(operatorID string) *K8sEventsConfig {
 	return &K8sEventsConfig{
-		InputConfig: helper.NewInputConfig(operatorID, "k8s_event_input"),
+		InputConfig:        helper.NewInputConfig(operatorID, "k8s_event_input"),
+		Namespaces:         []string{},
+		DiscoverNamespaces: false,
```

Comment:
Any reason we default to false _and_ empty? Seems like, in most cases, people will just want to collect all

Reply:
Good catch. I had originally set this default to true, but then later switched it to false thinking that people wouldn't like that default. I can switch it back.